src/OrchardCore.Modules/OrchardCore.Tenants/Controllers/AdminController.cs

```diff
@@ -554,21 +554,24 @@ private bool IsDefaultShell()
 
         private async Task UpdateShellSettingsAsync(ShellSettings settings, string action)
         {
+            string message;
+
             switch (settings.State)
             {
                 case TenantState.Running:
-                    _notifier.Warning(H["The tenant '{0}' is already {1}.", settings.Name, action]);
+                    message = String.Concat("The tenant '{0}' is already ", action, ".");
```

Comment:
You can't localize a dynamic string. Comment applies to all subsequent usages too

Reply:
Strange, I expect all string will be evaluated before the `message` it passed in the localizer

Reply:
Strange, I expect all string will be evaluated before the `message` it passed in the localizer

Reply:
Sure, but the string is still dynamically created. And there is no way to extract the possible texts in pot files from the localization extractor. 
Good: `H["some string {0}"]
Bad: `H[variable]`